### DISCUSSION TITLE
Add Support for Camera Imilab EC2

### DISCUSSION THREAD

**Discussion by perseus177** [XiaoMi/ha_xiaomi_home#487] [Ideas]
Please, Add support for camera Imilab EC2 
![Screenshot_20241228_194118_Mi Home](https://github.com/user-attachments/assets/a306c328-bb82-41fa-95ea-d062f6a48dd5)

**Comment by SusanPhevos** [comment]
这个设备太老了，厂商没有定义有效的 MIoT-Spec-V2.

**Reply by SusanPhevos** [reply]
The device is too old. The manufacturer did not define a valid MIoT-Spec-V2 for it.
